docs/src/main/asciidoc/websockets.adoc

```diff
@@ -45,8 +45,7 @@ First, we need a new project. Create a new project with the following command:
 mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
     -DprojectGroupId=org.acme \
     -DprojectArtifactId=websockets-quickstart \
-    -Dextensions="undertow-websockets" \
-    -DnoExamples
+    -Dextensions="undertow-websockets"
```

Comment:
Are we sure we want to do that? It won't be in the way of the rest of the guide?

Reply:
@ia3andy could you check if this makes sense. My understanding is that it's not really a good idea to have the examples in the way of the guides.

Reply:
As far I can see, there are more guides without `-DnoExamples` that with this property. One example: https://quarkus.io/guides/rest-json
Anyway, let me know if you want this change reverted if you prefer this way.

Reply:
@Sgitario could you revert this change, we decided not to generate the solution in the guides. At some point we will sync the quickstart by generating it from the codestart (for those which are iso) and give multiple way to get the solution (cmd, quickstart repo, code.quarkus.io) in the Solution section https://quarkus.io/guides/websockets#solution.

Reply:
Reverted